### PR TITLE
More updates to hopper example

### PR DESCRIPTION
**Pull request by tkuchida**
Updates carried over from #1075, which was polluted with large gifs.
[ci skip]

### CODE REVIEW

**Comment by chrisdembia** [comment]
I really like the README.

**Comment by chrisdembia** [comment]
I really like how the first and third gifs are in sync with each other.

**Comment by aseth1** [comment]
Cool! I can put the gifs in the Intro to the example! Hope you don't mind.

**Comment by tkuchida** [comment]
> I can put the gifs in the Intro to the example! Hope you don't mind.

I didn't know you would be presenting part of the example. Let me know if you want higher resolution gifs.

**Comment by aseth1** [comment]
I wasn't sure what sort of transition if any you had planned so I thought I should be ready to do it. I only have one slide with some background info. You can take it from there. Do you have slides you want to add or take it away on the board?

**Comment by tkuchida** [comment]
> one slide with some background info.

Perhaps just use the README.md on GitHub?

**Comment by tkuchida** [comment]
Only minor updates to the hopper example; merging.

File: OpenSim/Sandbox/ExampleHopperDevice/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+Example: *Hopper Device* 
+========================
+
+This example demonstrates some of the new features of the OpenSim 4.0 API.
+The Component architecture allows us to join sub-assemblies to form larger Models, with information flowing between Components via Inputs, Outputs, and Connectors.
+For more information, please refer to the [OpenSim doxygen](http://doxygen.opensim.community/) for the Component class.
+
+This interactive example consists of three steps:
+
+| []()                                                                   | []()
```
Comment (chrisdembia) [inline OpenSim/Sandbox/ExampleHopperDevice/README.md]: This is a really good idea!

Comment (chrisdembia) [inline OpenSim/Sandbox/ExampleHopperDevice/README.md]: Also, interestingly, it is similar to the notation for lambdas in C++: `[](){}`.

**Pull request merged**